### PR TITLE
csv output and fg logic

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

**Comment by jamiehannaford** [comment]
:+1: LGTM. just need to add the extra cols and the output line

**Comment by jamiehannaford** [comment]
we can probably centralize a lot of the iteration logic - but that's really not important right now and can be saved for later :fireworks:

**Comment by smashwilson** [comment]
Looks great to me.

Can you rebase to deal with the conflicts?

**Comment by jrperritt** [comment]
Nah, just merge as is ;)

**Comment by jrperritt** [comment]
Conflicts resolved

File: main.go [hunk 1]
```diff
@@ -42,15 +41,11 @@ func main() {
 
 	regions, fg := Regions(provider, opts)
 
-	fmt.Printf("Regions with a compute endpoint: %s.\n", strings.Join(regions, ", "))
+	fmt.Printf("Regions with a compute endpoint: %#v\n", regions)
```
Comment (smashwilson) [inline main.go]: This should be resolved the other way!

File: main.go [hunk 2]
```diff
@@ -42,15 +41,11 @@ func main() {
...
 	if fg {
 		fmt.Println("You do have a first-gen endpoint, too.")
```
Comment (jamiehannaford) [inline main.go]: also, do we need this line anymore? @smashwilson @jrperritt

File: main.go (hunk 1 above)
Comment (jrperritt) [inline main.go]: Fixed [here](https://github.com/jrperritt/cs-reboot/commit/179f96381f7b72883fdf46e6aa9ca51c71bd8b96)

File: main.go (hunk 2 above)
Comment (smashwilson) [inline main.go]: It shouldn't hurt. It goes along with the region banner as providing "and here's what I found in your service catalog" information.

Comment (jamiehannaford) [inline main.go]: ok, let's merge it

**Comment by smashwilson** [comment]
:rocket:

**Pull request merged**